client/FxAccountClient.js

```diff
@@ -1227,6 +1235,67 @@ define([
   };
 
   /**
+   * Get all the recovery emails associated with user that belongs to the
+   * session.
+   *
+   * @method recoveryEmails
+   * @param {String} sessionToken SessionToken obtained from signIn
+   */
+  FxAccountClient.prototype.recoveryEmails = function (sessionToken) {
+    required(sessionToken, 'sessionToken');
+
+    var request = this.request;
+    return hawkCredentials(sessionToken, 'sessionToken',  HKDF_SIZE)
+      .then(function(creds) {
+        return request.send('/recovery_emails', 'GET', creds);
+      });
+  };
+
+  /**
+   * Create a new email for account that belongs to this passed session.
+   *
+   * @method createEmail
+   * @param {String} sessionToken SessionToken obtained from signIn
+   * @param {String} email new email to be added
+   */
+  FxAccountClient.prototype.createEmail = function (sessionToken, email) {
+    required(sessionToken, 'sessionToken');
+    required(sessionToken, 'email');
+
+    var data = {
+      email: email
+    };
+
+    var request = this.request;
+    return hawkCredentials(sessionToken, 'sessionToken',  HKDF_SIZE)
+      .then(function(creds) {
+        return request.send('/recovery_email', 'POST', creds, data);
+      });
+  };
+
+  /**
+   * Remove the email address from the user account
+   *
+   * @method deleteEmail
+   * @param {String} sessionToken SessionToken obtained from signIn
+   * @param {String} email email to be removed
+   */
+  FxAccountClient.prototype.deleteEmail = function (sessionToken, email) {
+    required(sessionToken, 'sessionToken');
+    required(sessionToken, 'email');
+
+    var data = {
+      email: email
+    };
+
+    var request = this.request;
+    return hawkCredentials(sessionToken, 'sessionToken',  HKDF_SIZE)
+      .then(function(creds) {
+        return request.send('/recovery_email/destroy', 'POST', creds, data, {});
```

Comment:
What is the 5th option to `send`, the empty `{}` - other POST requests don't send it, is it necessary?